packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -64,6 +64,7 @@ library TaikoData {
         bytes32 parentMetaHash;
         HookCall[] hookCalls; // DEPRECATED, value ignored.
         bytes signature;
+        uint32 l1StateBlockNumber;
```

Comment:
As this will break our node in mainnet, lets merge this code and other potential changes to a preconf branch so that main branch can still be used to upgrade mainnnet.